src/main/java/uk/gov/hmcts/divorce/divorcecase/model/CaseData.java

```diff
@@ -110,6 +110,13 @@
     @Builder.Default
     private GeneralReferral generalReferral = new GeneralReferral();
 
+    @CCD(
+        label = "General Referrals",
+        typeOverride = Collection,
+        typeParameterOverride = "GeneralReferral"
+    )
+    private List<ListValue<GeneralReferral>> generalReferrals;
```

Comment:
are we displaying this on General Referral tab ?

Reply:
The tab update will be another PR